extmod/uasyncio/core.py

```diff
@@ -228,6 +239,24 @@ def run_until_complete(self, aw):
     def close(self):
         pass
 
+    def set_exception_handler(self, handler):
+        global _exc_handler
+        _exc_handler = handler
+
+    def get_exception_handler(self):
+        return _exc_handler if _exc_handler is not None else self.default_exception_handler
```

Comment:
By initialising `_ext_handler` like this:
```python
_exc_handler = Loop.default_exception_handler
```
this function can simply return `_exc_handler`.